README.md

```diff
@@ -159,7 +159,7 @@ Once NGINX Plus is deployed you can access the dashboard on port `http://<hostna
 Next we're installing cert-manager. cert-manager will allow you to integrate the Venafi platforms with Kubernetes to issue TLS certificates and provide identity management for applications across your cluster
 To install cert-manager run:
 ```console
-$ kubectl apply -f "https://github.com/jetstack/cert-manager/releases/download/v0.14.1/cert-manager.yaml"
+$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.3/cert-manager.yaml
```

Comment:
--validate=false should no longer be needed, our docs do this because people are bad at updating kubectl

Reply:
Should this also be removed from the cert-manager docs (as this is a copy :) )?

Reply:
>  people are bad at updating kubectl

maybe not yet even when you look at people's kubectl versions they are quite behind.... but here we do install it ourselves

Reply:
Done.